src/plugins/pbjx/components/history/delegate.js

```diff
@@ -28,6 +34,24 @@ export default (dispatch) => ({
     }));
   },
 
+  handleRevert: (formName, selected, node) => {
+    selected.forEach((item) => {
+      const { id, value } = item;
+      if (id === 'blocks') {
+        const canvasBlocks = value.filter((x) => x !== null).map((x) => Message.fromObject(x));
+        const editorState = convertToEditorState(canvasBlocks);
+        dispatch(storeEditor(formName, editorState, true));
+      } else {
+        const fieldType = node.schema().getField(id).getType().typeName.name;
+        if (fieldType === 'DATE_TIME' && value) {
+          dispatch(change(formName, camelCase(id), new Date(value)));
```

Comment:
@rsumilang this doesnt work for live_m3u8_url because that camelCases to liveM3U8Url